Terminal.Gui/Views/LineView.cs

```diff
@@ -71,6 +71,7 @@ public LineView (Orientation orientation)
 		public override void Redraw (Rect bounds)
 		{
 			base.Redraw (bounds);
+			bounds = Bounds;
```

Comment:
Should be able to revert this change once https://github.com/gui-cs/Terminal.Gui/pull/2309 is merged.